doc/source/reference/distutils.rst

```diff
@@ -214,102 +214,6 @@ template and placed in the build directory to be used instead. Two
-it will be replaced with the <index>-th word in the same repeat
-specification. There are two forms for the repeat rule: named and
-short.
-
-
-Named repeat rule
-^^^^^^^^^^^^^^^^^
-
-A named repeat rule is useful when the same set of repeats must be
-used several times in a block. It is specified using <rule1=item1,
-item2, item3,..., itemN>, where N is the number of times the block
-should be repeated. On each repeat of the block, the entire
-expression, '<...>' will be replaced first with item1, and then with
-item2, and so forth until N repeats are accomplished. Once a named
-repeat specification has been introduced, the same repeat rule may be
-used **in the current block** by referring only to the name
-(i.e. <rule1>.
-
-
-Short repeat rule
-^^^^^^^^^^^^^^^^^
-
-A short repeat rule looks like <item1, item2, item3, ..., itemN>. The
-rule specifies that the entire expression, '<...>' should be replaced
-first with item1, and then with item2, and so forth until N repeats
-are accomplished.
-
-
-Pre-defined names
-^^^^^^^^^^^^^^^^^
-
-The following predefined named repeat rules are available:
-
-- <prefix=s,d,c,z>
-
-- <_c=s,d,c,z>
-
-- <_t=real, double precision, complex, double complex>
-
-- <ftype=real, double precision, complex, double complex>
-
-- <ctype=float, double, complex_float, complex_double>
-
-- <ftypereal=float, double precision, \\0, \\1>
-
-- <ctypereal=float, double, \\0, \\1>
-
-
-Other files
------------
-
-Non-Fortran files use a separate syntax for defining template blocks
-that should be repeated using a variable expansion similar to the
-named repeat rules of the Fortran-specific repeats. The template rules
-for these files are:
-
-1. "/\**begin repeat "on a line by itself marks the beginning of
-   a segment that should be repeated.
-
-2. Named variable expansions are defined using #name=item1, item2, item3,
-   ..., itemN# and placed on successive lines. These variables are
-   replaced in each repeat block with corresponding word. All named
-   variables in the same repeat block must define the same number of
-   words.
-
-3. In specifying the repeat rule for a named variable, item*N is short-
-   hand for item, item, ..., item repeated N times. In addition,
-   parenthesis in combination with \*N can be used for grouping several
-   items that should be repeated. Thus, #name=(item1, item2)*4# is
-   equivalent to #name=item1, item2, item1, item2, item1, item2, item1,
-   item2#
-
-4. "\*/ "on a line by itself marks the end of the variable expansion
-   naming. The next line is the first line that will be repeated using
-   the named rules.
-
-5. Inside the block to be repeated, the variables that should be expanded
-   are specified as @name@.
-
-6. "/\**end repeat**/ "on a line by itself marks the previous line
-   as the last line of the block to be repeated.
+.. _here: http://www.numpy.org/devdocs/reference/distutils_guide.html#template-files
```

Comment:
Because the heading changed, the anchor is no longer `template-files` but `conversion-of-src-files-using-templates`

Reply:
And it should not be a full link, rather replace the `here` above (line 217) with
```
Read more `here <conversion-of-src-files-using-templates>`
```